packages/wonder-blocks-button/components/button-core.js

```diff
@@ -76,6 +70,7 @@ export default class ButtonCore extends React.Component<Props> {
         const commonProps = {
             "aria-disabled": disabled ? "true" : undefined,
             "data-test-id": testId,
+            role: "button",
```

Comment:
Because we style the `Button` component to look like a button, even when the tag is `<a>`, we want the role to be `button` so users can expect that space also works as a way to trigger this.

Reply:
This is a really awesome improvement, thank you!